app/assets/stylesheets/custom.scss

```diff
@@ -47,6 +47,28 @@ h3 {
   min-height: 50px;
 }
 
+#flash-error {
+  color: #ff0000;
+  font-weight: bold;
```

Comment:
Properties should be ordered color, font-size, font-weight, margin-top, text-align